README.md

```diff
@@ -225,7 +225,12 @@ Example [app](https://github.com/phrase/android-sdk-example)
 If the wrong version of a translation is being used, ensure a release with the latest translations and the current app version is available and the `versionName` for the app set and are using the `<major>.<minor>.<point>`. format.
 
 
-## Known limitations
+## Limitations
 
 - Inflated menus and preferences are not supported yet
+- Texts in custom views need to be updated [manually](https://github.com/phrase/phrase-android/?tab=readme-ov-file#custom-view-support)
+- Kotlin Multiplatform is not officially supported
+- Jetpack Compose views need to be wrapped with `Phrase { ... }.`
+- Plural translations are not yet supported with Jetpack Compose
+- Supported Android versions: API level 21+
 - Some libraries that do not support automatic context unwrapping can cause issues. For example `Hilt` requires to disable automatic context wrapping in Compose components https://github.com/phrase/phrase-android/releases/tag/3.5.0
```

Comment:
@theSoenke this link seems a bit odd. What does this refer to?

Reply:
Generally it's still correct that some libraries cause issues when the change the Android context. I've restructured it to make it clear it's only an issue when using Jetpack Compose and linked another section in the readme itself how to disable wrapping components